crates/api_models/src/payments.rs

```diff
@@ -1355,12 +1360,14 @@ pub struct PaymentsResponse {
     /// The business_sub_label for this payment
     pub business_sub_label: Option<String>,
 
-    /// Allowed Payment Method Types for a given PaymentIntent
-    #[schema(value_type = Option<Vec<PaymentMethodType>>)]
-    pub allowed_payment_method_types: Option<Vec<api_enums::PaymentMethodType>>,
-
     /// ephemeral_key for the customer_id mentioned
     pub ephemeral_key: Option<EphemeralKeyCreateResponse>,
+
+    /// the metadata that will be deprecated
+    pub metadata: Option<MetadataExternal>,
+
+    /// user defined values
+    pub udf: Option<pii::SecretSerdeValue>,
```

Comment:
Why are these fields not included in API schema

Reply:
`metadata` would be deprecated so we don't want it in API schema. Will add udf.